ccd-definitions-jsonnet/CaseEventToFields.jsonnet

```diff
@@ -0,0 +1,24 @@
+local base = import 'lib/base.libsonnet';
+local constants = import 'lib/constants.libsonnet';
+
+local CaseEventToField(CaseEventID, CaseFieldID, DisplayContext, CallBackURLMidEvent = '', PageFieldDisplayOrder = 1) = base.LiveDates + {
```

Comment:
We can also have assertions / throw errors e.g.

```
assert PageFieldDisplayOrder > 0 : 'Display order cannot be negative',
```

It will report the following during transpiration:

```
RUNTIME ERROR: Display order cannot be negative
	CaseEventToFields.jsonnet:5:38-72	thunk <object_assert>
	CaseEventToFields.jsonnet:22:5-118	thunk <array_element>
	During manifestation
```